downstream.js

```diff
@@ -19,7 +19,7 @@ const replacements = {
     type: 'simple',
     mappings: {
       // Order matters: specific URLs first to prevent partial matches
-      'https://docs.kuadrant.io/latest/kuadrant-operator/doc/user-guides/secure-protect-connect-single-multi-cluster/':
+      'https://docs.kuadrant.io/latest/kuadrant-operator/doc/user-guides/full-walkthrough/secure-protect-connect-openshift/':
```

Comment:
I think you'll also have to update the link in [links.ts](https://github.com/Kuadrant/kuadrant-console-plugin/blob/main/src/constants/links.ts#L22)

Reply:
good spot, forgot about that one